src/localization.js

```diff
@@ -101,11 +101,18 @@ angular.module('ngLocalize')
                                 deferrences[path].resolve(path);
                             }
                         })
-                        .error(function () {
+                        .error(function (err) {
+                            var path = getPath(token);
```

Comment:
Path can be an array due to https://github.com/doshprompt/angular-localization/blob/feat-72/src/localization.js#L63